docs/en/edge/guide/platforms/android/tools.md

```diff
@@ -85,6 +85,43 @@ AVD is available as a target, you're prompted to select one. By
 default the `run` command detects a connected device, or a currently
 running emulator if no device is found.
 
+## Signing the App
+
+You can review Android app signing requirements here: http://developer.android.com/tools/publishing/app-signing.html
+
+To sign an app, you need the following parameters:
+  * Keystore (`--keystore`): Path to a binary file which can hold a set of keys.
+  * Keystore password (`--storePassword`): Password to the keystore
+  * Alias (`--alias`): The id specifying the private key used for singing.
+  * Password (`--password`): Passsword for the private key specified.
+  * Type of the keystore (`--keystoreType`): pkcs12, jks (Default:pkcs12)
```

Comment:
believe the default is jks. When not specified, it'll switch to pkcs12 automatically if the keystore path ends with ".p12" or ".pfx", so you normally wouldn't need to set this explicitly.